src/editor/gridstack/gridstackWidget.ts

```diff
@@ -253,10 +244,10 @@ export class GridstackWidget extends Widget {
     items.forEach(el => {
       this._model.setCellInfo(el.id as string, {
         hidden: false,
-        col: el.x!,
-        row: el.y!,
-        width: el.width!,
-        height: el.height!
+        col: el.x || 0,
```

Comment:
I guess this could also be written with the `??` operator?

Reply:
Even though in this case it should be pretty similar since these fields are numbers.

Reply:
Doing a commit

Reply:
:+1: